Quality_Declaration.md

```diff
@@ -0,0 +1,179 @@
+
+There is an automated test which runs a linter that ensures each file has a license statement. [Here](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/spdlog_vendor/) can be found a list with the latest results of the various linters being run on the package.
+
+### Copyright Statements [3.iv]
+
+The copyright holders each provide a statement of copyright in each source code file in `spdlog_vendor`.
+
+There is an automated test which runs a linter that ensures each file has at least one copyright statement. Latest linter result report can be seen [here](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/spdlog_vendor/copyright/).
+
+## Testing [4]
+
+### Feature Testing [4.i]
+
+`spdlog_vendor` does not have feature testing. For vendored packages, this will be considered to be addressed in the external dependency.
+
+### Public API Testing [4.ii]
+
+`spdlog_vendor` does not have Public API testing. For vendored packages, this will be considered to be addressed in the external dependency.
+
+### Coverage [4.iii]
+
+`spdlog_vendor` does not provide coverage testing. For vendored packages, this will be considered to be addressed in the external dependency.
+
+### Performance [4.iv]
+
+`spdlog_vendor` does not provide performance testing. For vendored packages, this will be considered to be addressed in the external dependency.
+
+### Linters and Static Analysis [4.v]
+
+`spdlog_vendor` uses and passes all the ROS2 standard linters and static analysis tools as described in the [ROS 2 Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/#linters-and-static-analysis). Passing implies there are no linter/static errors when testing against CI of supported platforms.
+
+Currently nightly test results can be seen here:
+* [linux-aarch64_release](https://ci.ros2.org/view/nightly/job/nightly_linux-aarch64_release/lastBuild/testReport/spdlog_vendor/)
+* [linux_release](https://ci.ros2.org/view/nightly/job/nightly_linux_release/lastBuild/testReport/spdlog_vendor/)
+* [mac_osx_release](https://ci.ros2.org/view/nightly/job/nightly_osx_release/lastBuild/testReport/spdlog_vendor/)
+* [windows_release](https://ci.ros2.org/view/nightly/job/nightly_win_rel/lastBuild/testReport/spdlog_vendor/)
+
+## Dependencies [5]
+
+### Direct and Optional Runtime ROS Dependencies [5.i]/[5.ii]
+
+`spdlog_vendor` does not have direct/optional runtime ROS dependencies.
+
+### Direct Runtime non-ROS Dependency [5.iii]
+
+`spdlog_vendor` depends directly on the external dependency `spdlog`, which is qualified as quality level 4 in its [Quality Declaration](./spdlog_Q_DECLARATION.md).
```

Comment:
Link will be wrong for external QD. https://github.com/ros2/spdlog_vendor/pull/8